specification/mariadb/resource-manager/readme.md

```diff
@@ -26,7 +26,7 @@ These are the global settings for the MariaDB API.
 
 ``` yaml
 title: MariaDBManagementClient
-description: MariaDB Client
+description: 'The Microsoft Azure management API provides create, read, update, and delete functionality for Azure MariaDB resources including servers, databases, firewall rules, VNET rules, log files and configurations with new business model.'
```

Comment:
quotes are not needed

Reply:
Also, why not similar changes for mariadb json (using the common parameters) as below?

Reply:
removed quotes. mariadb json already uses common types.